src/style.css

```diff
@@ -2,0 +1,37 @@
+html {
+  display: flex;
+  justify-content: center;
+}
+
+body {
+  margin: 0;
+}
+
+.stars__box {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../src/images/star-icon.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child){
+  margin: 0 4px 4px 0;
+}
+
+.stars--1:nth-child(2) .stars__star:nth-child(-n + 1 ),
+.stars--2:nth-child(3) .stars__star:nth-child(-n + 2),
+.stars--3:nth-child(4) .stars__star:nth-child(-n + 3),
+.stars--4:nth-child(5) .stars__star:nth-child(-n + 4),
+.stars--5:nth-child(6) .stars__star:nth-child(-n + 5) {
+  background-image: url('../src/images/star-filled.svg');
+}
```

Comment:
The `nth-child` pseudo-class usage here seems incorrect and overly complex. It looks like you're trying to select stars within differently rated `.stars` elements, but the `nth-child` is not necessary unless there's a specific reason for its usage that's not apparent in the CSS alone. Simplify the selectors if possible, and ensure they match the actual HTML structure.